templates/order/create_password.html

```diff
@@ -8,38 +8,32 @@
 
 
 {% block content %}
-  <div class="row thankyoupage">
+  <div class="row checkout__thankyoupage">
     <div class="col-xl-10 offset-xl-1">
-      <img class="thankyoupage__pirate-mail" src="{% static 'images/pirate-mail.png' %}" srcset="{% static 'images/pirate-mail.png' %} 1x, {% static 'images/pirate-mail2x.png' %} 2x">
+      <img class="checkout__thankyoupage__parrot" src="{% static 'images/parrot.png' %}" srcset="{% static 'images/parrot.png' %} 1x, {% static 'images/parrot2x.png' %} 2x">
       <h1>{% trans "Thank you for your" %}  <a href="{{ order.get_absolute_url }}">{% trans "order" %}</a>!</h1>
       <p>{% trans "We’ve sent a confirmation email with details to" %} <span>{{ email }}</span>.<br>
         {% trans "In any problems and questions contact us." %}</p>
-      <div class="row thankyoupage__form">
-        <div class="col-md-6">
-          {% if login_form is not None %}
```

Comment:
Why this feature is removed?
It should stay there, if customer is already registered (email exists in the db) we can't show registration form because e-mail address has to be unique.